index.js

```diff
@@ -378,4 +378,17 @@ export const mentorshipTeam = [
 			},
 		],
 	},
+	{
+    name: 'Khang Huu Nguyen',
+    title: 'Student @ UMass Amherst',
+    hobbies: 'J-pop, Gaming, Anime',
+    avatar: 'https://media.licdn.com/dms/image/D4E03AQFvJxo3u961kQ/profile-displayphoto-shrink_400_400/0/1708669047854?e=1716422400&v=beta&t=EyfNJkgkzmLMgQv9iAOQgKNBffPSMjFob3Q2lnHU2UE',
```

Comment:
replace with this `https://res.cloudinary.com/khoa165/image/upload/v1711089916/viettech/huukhang.jpg`